timescale-cloud/create-a-service.md

```diff
@@ -1,178 +1,73 @@
-# Exploring Timescale Cloud
-
-Welcome to Timescale Cloud. Timescale Cloud is a a managed TimescaleDB service
-hosted in 75+ regions in AWS, Azure, or GCP. Powered
-by [TimescaleDB][timescale-features], you can create database instances in the
-cloud and automate many of your most common operational tasks. This allows you
-to spend more time focusing on your time-series workloads and less time worrying
-about database management.
-
-Before we start, let's review a few core concepts and phrases:
-
-- **Account**: Your Timescale Cloud account. You can register for a Timescale Cloud account on the [Timescale Cloud signup][sign-up] page.
-- **Project**: An empty Project is created for you automatically when you sign-up. Projects organize groups of Services, and have different billing settings. You can use Projects as a way to organize Services in your account and provide access to those Services with other users.
-- **Service**: A Service is an instance that corresponds to a cloud service provider tier (e.g., AWS Timescale-Pro-512-IO-Optimized). You can access all your Services from the 'Services' tab for a given Project.
-- **Database**: Databases are created within a Service. You can view and create a Database within a Service by selecting one of your Services, and then selecting the 'Databases' tab.
-- **Service Plans**: A Service Plan defines the configuration and level of database management that will be performed for a given TimescaleDB deployment.
-
-### Step 1: Sign up for Timescale Cloud
-
-Visit the [Timescale Cloud signup page][sign-up] and supply your name, email address, and password.
-
-Once you've submitted the information, verify your account by clicking on the link in the
-email you receive. If you don't receive this email shortly after submitting the form,
-check your spam folder.
-
-### Step 2: Create your first service
-
-After you complete account verification, you can visit the [Timescale Cloud portal][timescale-cloud-portal]
-and login with your credentials.
-
-You can create a new service by clicking on the `Create a new service` button.
-
-In the resulting screen, you'll see several options that enable you to choose:
-
-- **Service type**. Today, we support TimescaleDB and Grafana services.
-- **Cloud provider**. We support Amazon Web Services, Google Cloud, or Microsoft Azure.
-- **Cloud region**. We support most cloud regions offered by each cloud provider.
-- **Service plan**. We support many common configurations of CPU, memory, storage, backup, and nodes. *If you're still a Timescale Cloud trial user*, we recommend using the `Dev` plan as it will be most cost effective during your evaluation period.
+# About services
+An MST service is an instance that corresponds to a cloud service provider tier.
+You can access all your services from the `Services` tab within your projects.
+Before you begin, make sure you have [signed in to MST][sign-up] and created your account.
```

Comment:
```suggestion
A Timescale Cloud service is an instance that corresponds to a cloud service provider tier.
You can access all your services from the `Services` tab within your projects.
Before you begin, make sure you have [signed up to Timescale Cloud][sign-up] and created your account.
```

Not sure about whether it should be "signed in" or "signed up" above given context.

Reply:
No, I think you're right, it's "sign up".